src/api/app/models/concerns/staging_project.rb

```diff
@@ -244,14 +257,17 @@ def extract_missing_reviews(request, review)
     # Instead, we could have something like
     # who = review.by_group || review.by_user || review.by_project || review.by_package
 
+    extracted_missing_reviews = {}
     [:by_group, :by_user, :by_package, :by_project].each do |review_by|
```

Comment:
:thought_balloon: You can use the `reduce` here also, and then get rid of `extracted_missing_reviews`